test/setup/create-app.ts

```
@@ -17,19 +19,34 @@ afterAll(async () => {
   const db = await ephemeralGel();
 
   let app;
   try {
-    const moduleFixture = await Test.createTestingModule({
-      imports: [AppModule],
+    let builder = Test.createTestingModule({
+      imports: [AppModule, ...(imports ?? [])],
+      providers,
     })
       .overrideProvider(LevelMatcher)
       .useValue(new LevelMatcher([], LogLevel.ERROR))
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)